doc/source/index.rst

```diff
@@ -1,5 +1,13 @@
-Ansys Sphinx Theme documentation |version|
-##########################################
+Ansys Sphinx Theme documentation
+################################
+
+**Date**: |today| **Version**: |version|
+
+**Useful links**:
+
+`Source Repository <https://github.com/ansys/ansys-sphinx-theme>`_ |
+`Issues <https://github.com/ansys/ansys-sphinx-theme/issues>`_ |
+:ref:`ref-getting-started`
```

Comment:
Although some Python pages like <https://numpy.org/doc/stable/> have this, UI/UX did not include these links. Thus, we should remove them.